src/server/ldap/client.ts

```diff
@@ -0,0 +1,128 @@
+import * as ldap from 'ldapjs';
+import * as express from 'express';
+import {EventEmitter} from 'events';
+import {User} from '../../lib/model';
+
+interface ILdapConfig {
+    server: {
+        url: string,
+    };
+    base: string;
+    search: ldap.Client.SearchOptions;
+}
+let ldapConfig: ILdapConfig = {
```

Comment:
lets move this to the ldap.config.json file resided in the project root dir, see example [here](https://github.com/edu-xored/calendar/blob/master/src/server/database/database.ts#L3)